js/viz.js

```diff
@@ -45,22 +49,44 @@ const viz = {
       simulation.nodes(this.nodes);
     }
 
+    this.registerSimulationForces(simulation);
+
+    simulation.alphaTarget(1);
+    for (let i = 0; i < 10; i++) {
+      simulation.tick();
+    }
```

Comment:
This 10 constant seems pretty arbitrary. This should also be a descriptive constant somewhere.
However is there any way to keep ticking until there are no more changes?